aten/src/ATen/native/quantized/cpu/qsigmoid.cpp

```diff
@@ -76,6 +73,15 @@ Tensor qnnpack_sigmoid(Tensor input) {
     "failed to run QNNPACK sigmoid operator");
   return qy;
 }
+
+// This ALWAYS outputs scale=1.0/256, dtype=quint8
+// The zero_point is 0 for qint32 and quint8, but -128 for qint8.
+Tensor qnnpack_sigmoid(Tensor input) {
```

Comment:
Do we need this additional function? Can we directly call `qnnpack_sigmoid_observed_output` for qnnpack?

Reply:
oh sure